src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -1494,7 +1497,8 @@ export function makeScatterPlotData(
     copyNumberAlterations?:{
         molecularProfileIds:string[],
         data:AnnotatedNumericGeneMolecularData[]
-    }
+    },
+    selectedGene?:Gene
```

Comment:
I think this parameter name (`selectedGene`) needs to be clearer. is this specifically for treatments?

if so then the condition on line 1540 and 1541 should be specifically linked to treatments, not just if there are no genes selected.

Reply:
@adamabeshouse At the moment, yes this parameter is specifically for treatment vs clinical attribute plot. When calling the function in `PlotsTab`, I already added the condition `this.isClinicalAttrVsTreatment ? this.selectedGeneForTreatment : undefined`, so I thought it wasn't necessary to add condition inside the function's logic.

Reply:
ah okay. in that case, I believe the parameter name should definitely contain this information, named something like `selectedGeneForClinicalAttrVsTreatment`